livre/remote/dataSource.cpp

```diff
@@ -40,28 +41,28 @@ namespace livre
 {
 namespace remote
 {
-using boost::lexical_cast;
 
 static const uint32_t timeout = 60000; /*ms*/
 namespace
 {
-lunchbox::URI _getSinkURI( const DataSourcePluginData& initData )
+::zeq::URI _getSinkURI( const DataSourcePluginData& initData )
 {
-    const lunchbox::URI& initURI = initData.getURI();
-    lunchbox::URI::ConstKVIter i = initURI.findQuery( "bind" );
-    if( i == initURI.queryEnd( ))
-        return lunchbox::URI( "livresink://" );
-    return lunchbox::URI( "livresink://" + i->second );
+    const ::lunchbox::URI& initURI = initData.getURI( );
+    ::zeq::URI uri;
+    uri.setHost( initURI.getHost( ));
+    ::lunchbox::URI::ConstKVIter i = initURI.findQuery( "bind" );
+    if( i != initURI.queryEnd( ))
+        uri.setPort( boost::lexical_cast< uint16_t >( i->second ));
```

Comment:
Do we need the "bind" keyword in case of the new zeq::URI. It is in default host:port ?

livresink://./?bind=host:port will be tcp://host:port ?